api/users/serializers.py

```diff
@@ -34,6 +35,27 @@ def to_representation(self, value):
         return relationship_links
 
 
+class SocialField(ser.DictField):
```

Comment:
Wasn't sure about this approach -- it's not really a deprecated field, as the field is still there, just returns data in a slightly different format depending on the version, which I couldn't find another example of.